docker/simplified_app.sh

```diff
@@ -40,10 +41,9 @@ export LANG=C.UTF-8
 useradd -ms /bin/bash -U simplified
 
 # Get the proper version of the codebase.
-mkdir /var/www && cd /var/www
-git clone https://github.com/${repo}.git circulation
-chown simplified:simplified circulation
-cd circulation
+mkdir -p /var/www/circulation && rsync -rtv /ls_build/ /var/www/circulation
```

Comment:
instead of doing an rsync here, couldn't we copy the files directly into this path: 
COPY . /var/www/circulation